ffsubsync/ffsubsync.py

```diff
@@ -175,11 +186,11 @@ def run(args):
             else:
                 logger.error('failed to create test archive; no formats supported '
                              '(this should not happen)')
-                retval = 1
+                result['retval'] = 1
             logger.info('...done')
         finally:
             shutil.rmtree(tar_dir)
-    return retval
+    return result
```

Comment:
Github won't let me leave an inline comment on line 275 so leaving an inline comment here instead. Let's have `main()` return the retval instead of the full dictionary, so on line 275 change `return run()` to `return run()['retval']`. That way we'll provide the correct return value to `sys.exit(main())` if this is called as an entrypoint.

Reply:
I agree. It's now done.